addon/helpers/page-title.js

```diff
@@ -18,7 +35,9 @@ function updateTitle(tokens) {
  */
 export default Helper.extend({
   pageTitleList: service(),
-  headData: service(),
+  document: service('-document'),
+  fastboot: service(),
+  isFastBoot: reads('fastboot.isFastBoot'),
```

Comment:
I think we can't rely on `service:fastboot` being present. Applications that don't have `FastBoot` installed at all will throw an error here.

Reply:
Yes, that is correct. Why I ended up with this is for draft was that I wanted `ember test` to run both fastboot and browser tests in one go. Once I refactor this, those tests need to be run separately.

Reply:
Maybe make a `ember-try.js` scenario for `fastboot-tests` that adds the required dependencies? That way when `ember try:each` runs it will execute them (then you'd guard the `ember-cli-fastboot-testing` related tests for when `typeof FastBoot` isn't present)...

Reply:
Yes, we'll end up with that. 👍

Reply:
Scenario added.